server/game/cards/13-O4B/BayouVermilionRailroad.js

```diff
@@ -0,0 +1,39 @@
+const OutfitCard = require('../../outfitcard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class BayouVermilionRailroad extends OutfitCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            targetController: 'current',
+            effect: ability.effects.reduceFirstPlayedCardCostEachRound(1, card => card.hasKeyword('mystical'))
+        });
+        this.action({
+            title: 'Bayou Vermilion Railroad',
+            playType: 'noon',
+            cost: ability.costs.bootSelf(),        
+            target: {
+                activePromptTitle: 'Select a Mystical card',
+                cardCondition: { 
+                    location: 'play area',
+                    condition: card => card.hasKeyword('mystical') && card.isAtDeed() && !card.booted && card.canBeBooted()
+                }
+            },
```

Comment:
It is cleaner to use `gameAction` as that is used in the engine to check if the action is allowed and some other validation.
```suggestion
            target: {
                activePromptTitle: 'Select a Mystical card',
                cardCondition: { 
                    location: 'play area',
                    condition: card => card.hasKeyword('mystical') && card.isAtDeed()
                },
                gameAction: 'boot'
            },
```